### DISCUSSION TITLE
Configuration File

### DISCUSSION THREAD

**Discussion by coacharnold1** [navidrome/navidrome#2768] [Q&A] [answered]
Can someone post an example of a detailed configuration file with a little more specifics than the one on the wiki .    I'm little confused as to the syntax and the for the file takes..     What gets quotes and what doesn't?  as well the "ND_ " in front of everything,   is that just in the service file or in the configuration.toml file as well?      

for example:

is this correct sytax for the config file?:

LASTFM_APIKEY  "key here with quotes"
LASTFM_ENABLED = true

Thanks 

T

**Comment by metalheim** [comment, marked as answer]
Config file must be valid TOML Format, you can check the Syntax of TOML here https://toml.io/en/
And validate your config file in a TOML validator like 
 https://www.toml-lint.com/

The "ND_" prefixed Keys are never to be used in the config file. These are only used as environment variables

**Reply by coacharnold1** [reply]
Thanks .... perfect!!!!

**Reply by metalheim** [reply]
Did you manage to get it working?
If yes, please consider marking the Discussion as "solved" :)

**Comment by coacharnold1** [comment]
Solved
